src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -1001,6 +1006,34 @@ def __repr__(self):
         return repr_str
 
 
+class ConvertBox(tvt_v2.Transform):
+    """Convert bounding boxes to a different format."""
```

Comment:
Could you add docstring for arguments?

Reply:
removed this class https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/d5b77c306e41641e9616d5d3975f6b84d5e3f390